tests/testthat.R

```diff
@@ -1,4 +1,5 @@
 library(httptest)
+
```

Comment:
Why?

Reply:
I find the extra line there easier to read, but I've removed it.